runtime/cmd/cmd.go

```diff
@@ -74,18 +74,14 @@ var checkers = map[common.Location]*sema.Checker{}
 func DefaultCheckerInterpreterOptions(
 	checkers map[common.Location]*sema.Checker,
 	codes map[common.Location]string,
-	impls stdlib.FlowBuiltinImpls,
 ) (
 	[]sema.Option,
 	[]interpreter.Option,
 ) {
 
-	semaPredeclaredValues, interpreterPredeclaredValues :=
-		stdlib.FlowDefaultPredeclaredValues(impls)
+	// TODO: checker and interpreter base activations
```

Comment:
In a follow up PR it would be nice to refactor the command line runner and REPL to use the environment and avoid duplicating checker and interpreter creation/configuration.